config/main.py

```diff
@@ -287,13 +297,22 @@ def _abort_if_false(ctx, param, value):
         ctx.abort()
 
 def _stop_services():
-    services_to_stop = [
-        'swss',
-        'lldp',
-        'pmon',
```

Comment:
@stephenxs no need to have a separate service list. Just check the pmon presence and remove it only for Mellanox platforms.